src/spec/student_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../student'
+require_relative '../classroom'
+
+describe Student do
+  context 'When providing student information' do
+    classroom = Classroom.new('Microverse')
+    person = Student.new(id: nil, classroom: classroom, age: 22, name: 'leo', parent_permission: true)
+
+    it 'creates a student person' do
+      expect(person.name).to eq 'leo'
+    end
+
+    it 'adds student to classroom' do
+      expect(person.classroom).to eq classroom
+    end
+
+    it 'should print custom message' do
+      expect(person.play_hooky).to eq "¯\(ツ)/¯"
+    end
+  end
```

Comment:
- [ ] try to write some more tests, for example for students you can test for `parent_permission`, how the app reacts when it's true and when it's false.